test/fixtures/config/program-flows.js

```diff
@@ -7,8 +7,15 @@ module.exports = {
       id: faker.datatype.uuid(),
       name: `${ faker.company.bsBuzz() } ${ faker.company.catchPhraseNoun() }`,
       details: faker.lorem.sentences(),
-      published: faker.datatype.boolean(),
-      behavior: faker.random.arrayElement(['standard', 'conditional']),
+      published_at: faker.random.arrayElement([faker.date.between(
+        dayjs().subtract(2, 'week').format(),
+        dayjs().subtract(1, 'week').format(),
+      ), null]),
+      archived_at: faker.random.arrayElement([faker.date.between(
+        dayjs().subtract(2, 'week').format(),
+        dayjs().subtract(1, 'week').format(),
+      ), null]),
+      behavior: faker.random.arrayElement(['standard', 'automated']),
```

Comment:
My understanding is that program flows can't have the `Conditional` behavior. So I replaced that with the `Automated` behavior in the program flow fixture.

My understanding of which behaviors they can have:

1. Program flows: `Standard` or `Automated`
2. Program actions: `Standard`, `Automated`, or `Conditional` (but only if the action belongs to the flow)